torch/csrc/distributed/c10d/init.cpp

```diff
@@ -973,16 +973,57 @@ that adds a prefix to each key inserted to the store.
 #endif
 
   shared_ptr_class_<::c10d::ProcessGroup::Work>(module, "Work")
-      .def("is_completed", &::c10d::ProcessGroup::Work::isCompleted)
-      .def("is_success", &::c10d::ProcessGroup::Work::isSuccess)
-      .def("exception", &::c10d::ProcessGroup::Work::exception)
-      .def("source_rank", &::c10d::ProcessGroup::Work::sourceRank)
+      .def(
+          "is_completed",
+          [](::c10d::ProcessGroup::Work& work) -> bool {
+            TORCH_WARN_ONCE("ProcessGroup::Work::is_completed API is being "
+                "deprecated, please ping "
+                "https://github.com/pytorch/pytorch/issues/46291 "
+                "if you see this warning");
```

Comment:
Is this what clang-format gives you? The devserver clang-format gives me this:

```c++
            TORCH_WARN_ONCE(
                "ProcessGroup::Work::is_completed API is being "
                "deprecated, please ping "
                "https://github.com/pytorch/pytorch/issues/46291 "
                "if you see this warning");
```

Reply:
And ditto for other APIs as well.

Reply:
reformatted via vscode clang-format